dal/redis.go

```diff
@@ -0,0 +1,57 @@
+package dal
+
+import (
+	"fmt"
+	"github.com/go-redis/redis"
+)
+
+type Redis struct{
+	Client *redis.Client
+}
+
+func (r *Redis)InitClient(){
```

Comment:
please add space between 
```suggestion
func (r *Redis) InitClient() {
```